tests/utils/constants.py

```diff
@@ -1,5 +1,7 @@
 CHAIN_ID = 1263227476  # KKRT (0x4b4b5254) in ASCII
 
+DEPLOY_FEE = 10000
```

Comment:
I am not importing the constant from scripts here because if I do so it will ask for STARKNET_NETWORK environment variable.